### PR TITLE
Enable PHP 8.2 compat

### PR DESCRIPTION
**Pull request by koriym**

### CODE REVIEW

File: tests/Context/GetoptParserTest.php [hunk 1]
```diff
@@ -325,7 +325,7 @@ public function testParse_shortClusterOptional()
             '-b' => 'foo_arg',
         );
         $actual = $this->getopt_parser->getValues();
-        $this->assertSame($expect, $actual);
+        $this->assertContains($expect, $actual);
```
Comment (kenjis) [inline tests/Context/GetoptParserTest.php]: This seems a breaking change.

expected:
```
Array &0 (
    '-f' => true
    '-z' => true
    '-b' => 'foo_arg'
)
```
actual:
```
Array &0 (
    '-f' => true
    '-z' => true
    '-b' => true
    0 => 'foo_arg'
)
```

Comment (koriym) [reply tests/Context/GetoptParserTest.php]: Yes, it is. The -b is not set correctly.

But this also triggers an error in master.

Comment (kenjis) [reply tests/Context/GetoptParserTest.php]: Can you patch this?
https://github.com/auraphp/Aura.Cli/pull/60#issuecomment-894750519

Comment (koriym) [reply tests/Context/GetoptParserTest.php]: Wouldn't it be better to fix it there?

Comment (kenjis) [reply tests/Context/GetoptParserTest.php]: No. I'm going to send a PR soon.

Comment (kenjis) [reply tests/Context/GetoptParserTest.php]: I sent #67

Comment (koriym) [reply tests/Context/GetoptParserTest.php]: #67 is merged.

Comment (kenjis) [inline tests/Context/GetoptParserTest.php]: Please revert this change.

**Review by kenjis — APPROVED** [review]
